.github/workflows/verify-db-migration.yml

```
@@ -0,0 +1,271 @@
+
+      - name: Save test file from new commit
+        run: |
+          mkdir -p /tmp/test-scripts/helpers
+          cp Test/Altinn.Correspondence.Tests/verify_db_migration.js /tmp/test-scripts/ || true
+          cp Test/Altinn.Correspondence.UseCaseTests/helpers/correspondencePayloadBuilder.js /tmp/test-scripts/helpers/ || true
+
+      - name: Checkout repository at old commit
+        uses: actions/checkout@v6
+        with:
+          ref: ${{ needs.determine-commits.outputs.old_commit }}
+        path: ./Test/Altinn.Correspondence.Tests/old-commit
```

actionlint (1.7.10)

[error] unexpected key "path" for step to execute action. expected one of "continue-on-error", "env", "id", "if", "name", "timeout-minutes", "uses", "with"

(syntax-check)